CHANGES

```diff
@@ -72,6 +72,9 @@
      CMS_DEBUG_DECRYPT flag.
      [Bernd Edlinger]
 
+  *) The function NCONF_WIN32() has been deprecated.
```

Comment:
Could we expand this out more? Perhaps with a link to the man page

Reply:
sure, commit added.